docs/concepts/metrics.md

```diff
@@ -0,0 +1,85 @@
+---
+id: metrics
+title: Temporal Metrics
+description: Temporal and Workers both emit metrics| this page describes them
+sidebar_label: Metrics
+---
+
+## Worker/SDK Metrics
+
+The Temporal SDKs emit a largely common set of metrics. Note that all the below metrics are prefixed
+with `temporal_` before being exported to their configured destination(s). `Core`
+
+### Keys
+
+The metrics always have some keys/labels attached to them. Their definitions:
+
+- `task-queue` - Task queue the worker polls on
+- `namespace` - Namespace the worker is bound to
+- `poller_type` - One of `workflow_task`, `activity_task`, or `sticky_workflow_task`
+- `worker_type` - One of `ActivityWorker`, `WorkflowWorker`, or `LocalActivityWorker` (Go only)
+- `activity_type` - The function name / class name / etc of the activity the metric is associated with
+- `workflow_type` - The function name / class name / etc of the workflow the metric is associated with
+- `operation` - Only for client gRPC requests, is the name of the RPC method
+
+### Metrics definitions
+
+Note that `Core` includes the Typescript SDK.
+
+| Name                                      | Desc                                                                                                                                                            | Metric Type | Exists In SDK                               | Keys                                                |
```

Comment:
This table is a bit of a pain to maintain.
Also, I can't guarantee the info in this table is indexed and searchable.

My strong recommendation would be to convert each metric to its own h3 like we have done with these:
https://docs.temporal.io/docs/reference/events/
https://docs.temporal.io/docs/reference/commands/

However, if we want to keep the table, then we could do a hybrid approach like this:
https://docs.temporal.io/docs/go/how-to-set-startworkflowoptions-in-go

And limit the table columns and info -

Reply:
Like @sw-yx mentioned can we figure out if there's a way _to_ make it index-able? The table is *dramatically* more readable IMO. The hybrid is an OK hack, but it's still a hack at the end of the day. IMO a table is pretty clearly the right format for this information.

Reply:
Yes, there is a way to index tables. We will definitely look into it. But it hasn't been needed yet.
Thinking about this more - the real reason h3s are the best here is because 1 they provide us with the ability to link directly to the metric name itself. 2 we have a place to add more info per metric - tables do not support multi line content, and we can only have so many columns before it becomes just too much. I don't think of it as a hack. I think of it as a very strategic formatting decision. I wonder if the "tables" were turned (pun intended) and I came into your world of expertise and started calling all of your decisions "hacks" how you might respond to that?

Reply:
Sure, sorry, just my point is doesn't sound like you're saying the table is actually the wrong way to show this other than limitations imposed by the platform we're using. That's what I mean by a hack. If the limitations can be overcome, we would prefer to use a table.

If we can overcome the direct linking limitation, then the table is more compact and readable. As for adding more columns / content, sure, I can see breaking it out then, but the information needed here is unlikely to grow. If it does, we could break it out then.

Reply:
I mean, I think it is more about what is practical in the context of the given situation.
And right now, the context of the situation is this:

1. I don't plan on spending any time in the next few weeks messing with the Algolia config to support tables - perhaps someone else might have time to do it.
2. I don't know of any way atm to overcome the direct linking of table items - which is very handy when dropping links to users in support.
3. We do try to plan for information growth - there may be any number of things that we may want to add here.
4. The table I am looking at is IMO definitely NOT compact - it is huge, and width wise, doesn't fit on the page - It definitely doesn't fit mobile or tablet screens, which a decent percentage of traffic still happens to be.
5. A stand alone table doesn't match the format of the other references we have made.

So, I get that you don't want to do more work here.
But I think you can appreciate my desire to ensure this meets some level of "quality" up front and we aren't just adding more debt for us to go "clean up" later on.